Functions.Templates/Templates/DurableFunctionsEntityOrchestration-CSharp-2.x/DurableFunctionsEntityOrchestrationCSharp.cs

```diff
@@ -0,0 +1,48 @@
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Newtonsoft.Json.Linq;
+
+namespace Company.Function
+{
+    public static class DurableFunctionsEntityOrchestrationCSharp
+    {
+        [FunctionName("DurableFunctionsEntityOrchestrationCSharp_CounterHttpStart")]
```

Comment:
`DurableFunctionsEntityOrchestrationCSharp` would be replaced by the function name mentioned by the customer. Not sure if that would break the durable orchestration logic.

Reply:
It should be fine.